lib/absinthe/schema/notation.ex

```diff
@@ -709,10 +709,31 @@ defmodule Absinthe.Schema.Notation do
     |> record_private!(:meta, key, value)
   end
 
+  @doc """
+  Defines list of metadata's key/value pair for a custom type.
+  """
+  defmacro meta(keyword_list) do
+    __CALLER__
+    |> recordable!(:meta, @placement[:meta])
+    |> record_private!(:meta, keyword_list)
+  end
+
+  @doc false
+  # Record private values
+  def record_private!(env, raw_owner, raw_keyword_list) when is_list(raw_keyword_list) do
+    [owner, keyword_list] = Enum.map([raw_owner, raw_keyword_list], &Macro.expand(&1, env))
```

Comment:
You actually gotta do deep expansion here, so I'd just use the `expand/2` helper function:

```
owner = expand(owner, env)
keyword_list = expand(raw_keyword_list, env)
```

Reply:
But i am already using the expand macro ;)

Reply:
For reference, it did not come up with this myself, this construct was already being used:

https://github.com/absinthe-graphql/absinthe/blob/master/lib/absinthe/schema/notation.ex#L715